sections/rich-text.liquid

```diff
@@ -4,8 +4,35 @@
 <noscript>{{ 'component-rte.css' | asset_url | stylesheet_tag }}</noscript>
 <noscript>{{ 'section-rich-text.css' | asset_url | stylesheet_tag }}</noscript>
 
-<div{% unless section.settings.full_width %} class="page-width"{% endunless %}>
-  <div class="rich-text color-{{ section.settings.color_scheme }} gradient{% if section.settings.full_width %} rich-text--full-width{% endif %}">
+{% liquid
+  assign section_mobile_padding_top = section.settings.padding_top
+  assign section_mobile_padding_bottom = section.settings.padding_bottom
+
+  if section.settings.padding_top >= 28 or section.settings.padding_top <= 80
```

Comment:
These conditionals don't seem like they're working. I'm seeing negative paddings and 10px being subtracted from the value below 28px
<img src="https://screenshot.click/14-41-uqtsf-w1u6c.png" width="500" />

I think you might want an `and` instead of an `or`